tests/torch/fx/graph_manager.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import torch
+import torch.fx
+
+import nncf
+from nncf.common.factory import NNCFGraphFactory
+from nncf.common.graph.graph import NNCFGraph
+from nncf.common.graph.graph import NNCFNode
+from nncf.torch.model_graph_manager import OPERATORS_WITH_BIAS_METATYPES
+from nncf.torch.model_graph_manager import find_const_node_in_constant_subgraph
+
+
+def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
```

Comment:
@anzr299, @daniil-lyakhov, it looks like, this is general functions and can be used in the fast bias correction. Please consider possible refactoring to use functions in the NNCF algorithms.

Reply:
I have removed it for this PR for now. We can add it back in a follow up PR when refactoring